app/cafe.py

```
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import (
+    VaccineError, NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if "vaccine" not in visitor:
+                raise NotVaccinatedError("Visitor has no vaccine record.")
+            elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("Vaccine is expired.")
+
+            if not visitor.get("wearing_a_mask", False):
```

This violates the requirement: "If the visitor is not wearing a mask (wearing_a_mask is False), raise NotWearingMaskError." Currently the code uses `visitor.get("wearing_a_mask", False)`, which treats a missing key as False and raises the exception for visitors who simply don't have the key. Consider changing the check to only raise when the key exists and is explicitly False, for example:

`if "wearing_a_mask" in visitor and visitor["wearing_a_mask"] is False:
    raise NotWearingMaskError("Visitor is not wearing a mask.")`

This prevents KeyError and ensures absence of the key does not trigger the mask error.